### PR TITLE
Make langauge templates

### PR DESCRIPTION
**Pull request by delphiactual**
Allows for easy translation of the About and Support pages

### CODE REVIEW

**Comment by delphiactual** [comment]
This is complete, with the exception of not being able to fallback to english if a file is not found.
I have just put copies of the english version in the appropriate sub-directories, this has increased our filesize by 7kb per language, for a total extra of 35kb. If a fallback directive is necessary I will continue to look into it.

File: app/scripts/shell/dimAppCtrl.controller.js [hunk 1]
```diff
@@ -151,10 +171,10 @@
         }
       };
     }
-
+    var lang = ' ';
```
Comment (bhollis) [inline app/scripts/shell/dimAppCtrl.controller.js]: What's this (and the lang parameter) for?

File: app/scripts/shell/dimAppCtrl.controller.js [hunk 2]
```diff
@@ -124,21 +124,41 @@
...
+              name: name,
+              appendClassName: 'modal-dialog'
+            });
+            /* Lang Fallback
+            result = ngDialog.open({
+              template: 'views/en/' + name + '.html',
+              className: name,
+              name: name,
+              appendClassName: 'modal-dialog'
+            }); */
```
Comment (bhollis) [inline app/scripts/shell/dimAppCtrl.controller.js]: Remove

File: app/scripts/shell/dimAppCtrl.controller.js [hunk 3]
```diff
@@ -124,21 +124,41 @@
...
+          } else {
+            result = ngDialog.open({
+              template: 'views/' + name + '.html',
+              className: name,
+              name: name,
+              appendClassName: 'modal-dialog'
+            });
```
Comment (bhollis) [inline app/scripts/shell/dimAppCtrl.controller.js]: Rather than copy/pasting two blocks of code, just set `lang` to `''` if it's unset.

**Comment by kyleshay** [comment]
looks great, thanks @delphiactual. This about ready to be merged?

**Comment by delphiactual** [comment]
yes, changelog required? Added translations to About and Support?

**Comment by kyleshay** [comment]
Yup, please do!

**Comment by kyleshay** [comment]
Also, thank you @Korben85 !!

**Pull request merged**